lib/matplotlib/text.py

```diff
@@ -831,6 +846,7 @@ def get_position(self):
         # specified with 'set_x' and 'set_y'.
         return self._x, self._y
 
+    @_api.deprecated("3.5")
```

Comment:
```suggestion
    # When removing, also remove the hash(color) check in set_color()
    @_api.deprecated("3.5")
```

Without the note, we might miss that action.

Reply:
yup